pointers/pointers.c

```diff
@@ -51,9 +58,23 @@ void string_copy(char *x, char *y)
     
     Do not just use the `strcmp` function from the standard library.
 */
-int string_compare(char *m, char *n)
+char *find_string(char *haystack, char *needle)
 {
-
+    while (*haystack)
+    {
+        if (*haystack == *needle)
+        {
+            while (*needle)
+            {
+                if (*needle == *haystack)
+                {
+                    needle++;
+                }
+            }
+        }
+        haystack++;
+        needle++;
+    }
 }
 
```

Comment:
Hey Edwin, you seem to be making good progress with this, I highly encourage you to keep working on pointers!